dist/index.js

```diff
@@ -984,12 +987,20 @@ if (version) {
 }
 
 if (!buildNumber) {
+    execCommand(`agvtool what-version -terse`, {
+        listeners: { stdout: (data) => {
+            core.setOutput('build-number', +data + 1)
+        }}
+    })
```

Comment:
I think its better to do this at the end of this code so that you dont have to increment manually